Python/Product/Common/Strings.resx

```diff
@@ -1162,6 +1162,127 @@ Please refer to &lt;a href="https://aka.ms/pythononappservice"&gt;aka.ms/PythonO
   <data name="ConnErrorMessages_UnknownVersion" xml:space="preserve">
     <value>Unknown Python version loaded in process</value>
   </data>
+  <data name="RemoteUnrecognizedDebuggingTransport" xml:space="preserve">
+    <value>Unrecognized remote debugging transport '{0}'.</value>
+    <comment>{0} is the uri scheme</comment>
+  </data>
+  <data name="RemoteDebugPortSupplierDescription" xml:space="preserve">
+    <value>Allows debugging a Python process on a remote machine running any OS, if it can be connected to via TCP, and remote debugging has been enabled by using the 'ptvsd' module. Specify the secret, hostname and port to connect to in the 'Qualifier' textbox, e.g. 'tcp://secret@localhost:5678'. </value>
+  </data>
+  <data name="RemoteProcessAuthenticationErrorWarning" xml:space="preserve">
+    <value>{0}
+Connect anyway?</value>
+    <comment>{0} is error details</comment>
+  </data>
+  <data name="FailedToAttachDebugger" xml:space="preserve">
+    <value>Failed to attach debugger:
+{0}</value>
+    <comment>{0} is error details</comment>
+  </data>
+  <data name="DebugStackFrameParseTextError" xml:space="preserve">
+    <value>Error: {0}</value>
+    <comment>{0} is error message</comment>
+  </data>
+  <data name="DebugThreadUnknownLocation" xml:space="preserve">
+    <value>&lt;unknown location, not in Python code&gt;</value>
+  </data>
+  <data name="DebugTcpsTransportConnectionError" xml:space="preserve">
+    <value>Could not establish secure connection to {0} because of the following SSL issues:
+
+</value>
+    <comment>{0} is a URI
+A list of issues is concatenated to the message (one per line)</comment>
```

Comment:
I _think_ we should be adding {1} for the list of issues, or we may not handle RTL text properly (thinking specifically of the marker characters, if any, that may not behave consistently when blindly concatenating strings).